html/changelogs/AutoChangeLog-pr-30056.yml

```diff
@@ -0,0 +1,4 @@
+author: "More Robust Than You"
+delete-after: True
+changes: 
+  - rscdel: "Finally fucking removed gangs"
```

Comment:
wow robustin

Reply:
that one got into my master I really need to fix that